app-emacs/projectile/projectile-2.8.0-r1.ebuild

```diff
@@ -25,3 +25,11 @@ src_test() {
 
 	elisp-test
 }
+
+pkg_postinst() {
+	# Descriptions for this packages' purpose were taken from Projectile's
+	# home page https://docs.projectile.mx/projectile/usage.html
+	optfeature_header "Performance can be gained in ${PN} by installing the following packages:"
```

Comment:
Please limit line length to 80 positions.
Policy reference: https://devmanual.gentoo.org/ebuild-writing/file-format/index.html#indenting-and-whitespace

Reply:
Noted! Thanks for the feedback. 
I changed it in the new commit